### PR TITLE
Alignment batch

### PR DESCRIPTION
**Pull request by morsecodist**
# Description

This change removes the old ASG management stuff and adds some stuff that will be needed for the batch refactor so we can remove compatibility hacks.

# Tests

Ran various pipeline tests locally

Also tested on a branch of idseq-dag with no compatibility hacks

### CODE REVIEW

File: app/helpers/pipeline_runs_helper.rb [hunk 1]
```diff
@@ -65,10 +65,15 @@ def aegea_batch_submit_command(base_command,
                                  docker_image: "idseq_dag",
                                  sample_id: 0,
                                  stage_name: "misc")
+    # TODO: environment values once we use the new pipeline infra
+    #   that new infra uses dev rather than development
+    #   once we switch to that for the pipeline these values won't be needed
+    deployment_environment = Rails.env == "development" ? "dev" : Rails.env
```
Comment (tfrcarvalho) [inline app/helpers/pipeline_runs_helper.rb:71]: There is a mapping for in https://github.com/chanzuckerberg/idseq-web/blob/master/app/services/sfn_pipeline_dispatch_service.rb#L13. Please use the mapping but feel free to move it a more adequate place if necessary.

File: db/schema.rb [hunk 2]
```diff
@@ -502,8 +502,8 @@
     t.integer "taxid", null: false
     t.bigint "wikipedia_id"
     t.string "title"
-    t.text "summary"
-    t.text "description"
+    t.text "summary", limit: 16_777_215
```
Comment (cdebourcy) [inline db/schema.rb:505]: Are these unrelated schema changes intended? Was the schema file out of date before?

Comment (morsecodist) [reply db/schema.rb:505]: Sorry another weird mangling I will manually bring this back into line.

File: app/helpers/pipeline_runs_helper.rb [hunk 3]
```diff
@@ -65,10 +65,15 @@ def aegea_batch_submit_command(base_command,
                                  docker_image: "idseq_dag",
                                  sample_id: 0,
                                  stage_name: "misc")
+    # TODO: environment values once we use the new pipeline infra
```
Comment (davidrissato) [inline app/helpers/pipeline_runs_helper.rb:68]: Should we refer to a Jira ticket here?

File: app/models/pipeline_run.rb [hunk 4]
```diff
@@ -1246,9 +1170,9 @@ def load_chunk_stats
     return unless stdout
     outputs = stdout.split("\n").map { |line| line.split.last }
     gsnap_outputs = outputs.select { |file_name| file_name.start_with?("multihit-gsnap-out") && file_name.end_with?(".m8") }
-    rapsearch_outputs = outputs.select { |file_name| file_name.start_with?("multihit-rapsearch2-out") && file_name.end_with?(".m8") }
+    rapsearch2_outputs = outputs.select { |file_name| file_name.start_with?("multihit-rapsearch2-out") && file_name.end_with?(".m8") }
     self.completed_gsnap_chunks = gsnap_outputs.length
-    self.completed_rapsearch_chunks = rapsearch_outputs.length
+    self.completed_rapsearch2_chunks = rapsearch2_outputs.length
```
Comment (tfrcarvalho) [inline app/models/pipeline_run.rb:1175]: Do we still need this?

File: app/helpers/pipeline_runs_helper.rb (hunk 3 above)
Comment (morsecodist) [reply app/helpers/pipeline_runs_helper.rb:68]: I am not sure which one it is. This will eventually need to be removed completely. @kislyuk do we have a ticket to capture the decoupling work?

Comment (morsecodist) [reply app/helpers/pipeline_runs_helper.rb:68]: actually maybe I should just remove the TODO since instead of fixing it this section will just be deleted

Comment (tfrcarvalho) [reply app/helpers/pipeline_runs_helper.rb:68]: Should have a ticket anyway... If you find an existing one, great. Otherwise create a new one.

File: app/helpers/pipeline_runs_helper.rb (hunk 1 above)
Comment (tfrcarvalho) [reply app/helpers/pipeline_runs_helper.rb:71]: Since it is temporary, it is fine to use that location...

File: app/models/pipeline_run.rb [hunk 5]
```diff
@@ -126,25 +120,25 @@ class PipelineRun < ApplicationRecord
   STATUS_RUNNABLE = 'RUNNABLE'.freeze # TODO: (gdingle): not used anywhere?
   STATUS_READY = 'READY'.freeze
   # NOTE: The current stored job_status are...
-  # +-------------------------------------------+
-  # | job_status                                |
-  # +-------------------------------------------+
-  # | 1.Host Filtering-FAILED                   |
-  # | 1.Host Filtering-FAILED|READY             |
-  # | 2.GSNAPL/RAPSEARCH alignment-FAILED       |
```
Comment (gregdingle) [inline app/models/pipeline_run.rb:134]: this list is supposed to represent historical values, not current ones––if those are even changing

File: db/migrate/20200403194501_rapsearch_to_rapsearch2.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+class RapsearchToRapsearch2 < ActiveRecord::Migration[5.1]
+  def change
+    rename_column :pipeline_runs, :completed_rapsearch_chunks, :completed_rapsearch2_chunks
```
Comment (gregdingle) [inline db/migrate/20200403194501_rapsearch_to_rapsearch2.rb:3]: wow this is a pretty thorough rename!

in general, I'd be cautious about renaming any db col. you need to understand the downstream dependents really well, which could even include analytics reports! in this case, I think its safe

**Comment by gregdingle** [comment]
Red diffs are the best! 😄

File: db/migrate/20200403194501_rapsearch_to_rapsearch2.rb (hunk 6 above)
Comment (morsecodist) [reply db/migrate/20200403194501_rapsearch_to_rapsearch2.rb:3]: `completed_rapsearch_chunks` has no usages in the code base. I assume it was to do with the autoscaling but I am hesitant to drop it for historical reasons. I do want to ensure we rename it everywhere though.

File: app/models/pipeline_run.rb (hunk 4 above)
Comment (morsecodist) [reply app/models/pipeline_run.rb]: Nope, thanks!

**Comment by morsecodist** [comment]
Doing another test before I merge
